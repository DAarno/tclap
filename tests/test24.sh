#!/bin/sh
# failure
`dirname $0`/simple-test.sh `basename $0 .sh` test5 --aaa dilbert -b asdf -c fdas
