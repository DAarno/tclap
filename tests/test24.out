PARSE ERROR: Argument: --aaa AND -b provided.
             Only one is allowed.

Brief USAGE: 
   ../examples/test5  {-a string|-b string} {--eee string|--fff
                      string|-g string} {-i string ...|-j string
                      ...} [--ddd] -c string [--version] [-h]

For complete USAGE and HELP type: 
   ../examples/test5 --help

